lib/customer-deposit-wallet/rest/Cardano/Wallet/Deposit/REST/Start.hs

```diff
@@ -112,10 +114,19 @@ mockFundTheWallet network resource = flip runWalletResourceM resource $ do
     Right () <- liftIO $ postTx network tx
     pure ()
 
-newFakeBootEnv :: IO (WalletBootEnv IO)
-newFakeBootEnv =
-    WalletBootEnv
-        (show >$< stdoutTracer)
-        Read.mockGenesisDataMainnet
-        . mapBlock Read.EraValue
-        <$> newNetworkEnvMock
+newFakeBootEnv :: Maybe FilePath -> IO (WalletBootEnv IO)
+newFakeBootEnv genesisFile = do
+    eGenesisData <- runExceptT $ case genesisFile of
+        Nothing -> ExceptT $ pure $ Right Read.mockGenesisDataMainnet
+        Just file -> fst <$> Byron.readGenesisData file
+    print genesisFile
+    print eGenesisData
+    print $ Read.getNetworkId <$> eGenesisData
```

Comment:
Do we want to keep these debugging statements? 🤔

Reply:
For a bit, the `Start.hs` is going to be messed up with the real network environment anyway on next PR